app/views/dashboard/_restaurant.html.haml

```diff
@@ -4,6 +4,8 @@
       %h3.panel-title
         %a.collapse-link{role: 'button', data: {toggle: 'collapse'}, href: "#restaurant_#{restaurant.id}", aria: {expanded: !current_user.collapsed?, controls: "#restaurant_#{restaurant.id}"}, class: ('collapsed' if current_user.collapsed?)}
         = link_to restaurant.name, restaurant_path(restaurant)
+        = link_to '<i class="glyphicon glyphicon-list-alt"></i>'.html_safe, restaurant.menu_link, target: '_blank' if restaurant.menu_link.present?
```

Comment:
I think using block would be more readable here and on all other occasions
```
= link_to restaurant.menu_link, target: '_blank' do
  i.glyphicon.glyphicon-list-alt
```

Reply:
thanks.